### PR TITLE
SimpleSchedulerExtensions

### PR DESCRIPTION
**Pull request by Damian-P**
## Description
Allows a SenderContext to be a scheduler.

## Purpose
This pull request is a:

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by rogeralsing** [comment]
Nice!

File: src/Schedulers/Proto.Scheduler.SimpleScheduler/SimpleSchedulerExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Threading;
+
+namespace Proto.Schedulers.SimpleScheduler
+{
+    public static class SimpleSchedulerExtensions
+    {
+        public static ISimpleScheduler ScheduleTellOnce(this ISenderContext senderContext, TimeSpan delay, PID target, object message) =>
```
Comment (alexeyzimarev) [inline src/Schedulers/Proto.Scheduler.SimpleScheduler/SimpleSchedulerExtensions.cs:8]: I like these extensions, just have a suggestion to keep the code formatting aligned. That's why I included the `DotSettings` file to the repository. But it only works with Rider/RestSharp. If that's not enough, we could consider adding the editorconfig file.

Another suggestion is to pay attention to triple-slash comments for public APIs, so when we're ready to generate the API docs, everything gets properly documented.

Comment (Damian-P) [reply src/Schedulers/Proto.Scheduler.SimpleScheduler/SimpleSchedulerExtensions.cs:8]: Sorry, I'm using vscode with the default code formatter. You know how to adapt those settings to vscode ?

Comment (alexeyzimarev) [reply src/Schedulers/Proto.Scheduler.SimpleScheduler/SimpleSchedulerExtensions.cs:8]: VsCode only supports editorconfig but I am not sure how to export DotSettings to VsCode. I will fix it for you, no worries. Will check if it's not much work to create a proper editorconfig.

**Pull request merged**